### PR TITLE
Move the filtering of sensitive data to the config class

### PR DESCRIPTION
**Pull request by nickvergessen**
@MorrisJobke @DeepDiver1975

### CODE REVIEW

**Comment by PVince81** [comment]
Makes sense :+1:

File: lib/private/systemconfig.php [hunk 1]
```diff
@@ -28,6 +28,21 @@
...
+		'dbuser' => true,
+		'mail_smtpname' => true,
+		'mail_smtppassword' => true,
+		'passwordsalt' => true,
+		'secret' => true,
+		'ldap_agent_password' => true,
+		'objectstore' => ['arguments' => ['password' => true]],
+	];
+
+	const SENSITIVE_VALUE = '***REMOVED SENSITIVE VALUE***';
```
Comment (DeepDiver1975) [inline lib/private/systemconfig.php]: duplicate

**Comment by DeepDiver1975** [comment]
besides duplicate constant :+1:

File: lib/private/systemconfig.php (hunk 1 above)
Comment (nickvergessen) [inline lib/private/systemconfig.php]: fixed

**Comment by MorrisJobke** [comment]
:+1:

**Pull request merged**